project.yaml

```diff
@@ -890,7 +890,7 @@ Layer:
             CASE WHEN int_surface IN ('unpaved') THEN 0 ELSE 2 END
         ) AS roads_casing
     properties:
-      minzoom: 10
+      minzoom: 9
```

Comment:
It still used at z9 instead of *-fill layers.

Given that these layers are used anyway on z9 (solely to display highway=secondary halo) I am not sure what would result in the better performance.